### ISSUE TITLE
ACS SCT failure - SMBIOS

### ISSUE THREAD

**Issue by samerhaj** [pftf/RPi4#71]
On RPi4 Rel 1.15, running ACS 2.4:

https://raw.githubusercontent.com/pftf/acs-reports/master/v1.15/acs-enterprise-v2.4/SCT_RUN/Overall/Summary.log

```
UefiSmbios -- FAILURE
ED2399AE-416E-46C6-ACA2-D9B1CF9975CD
/home/edhcha01/RELEASE_V2.4/arm-enterprise-acs/luv/build/tmp/work/aarch64-oe-linux/sbbr/v1.1+gitAUTOINC+b558bad254-r0/edk2/SctPkg/TestCase/UEFI/EFI/Generic/SbbrSmbios/BlackBoxTest/SbbrSmbiosBBTestFunction.c:137 - SMBIOS30 table invalid version. 3.-549755813885
```

**Comment by samerhaj** [comment]
Re-ran with latest ACS built from upstream. Test passes.

Test is checking for minimum SMBIOS version of 3.1.1. The RPi4 is at 3.3, which is fine.

Closing

**Comment by samerhaj** [comment]
Confirmed issue is fixed in ACS 2.5 beta:

https://github.com/pftf/acs-reports/blob/master/v1.19/acs-enterprise-v2.5-beta/SCT_RUN/Overall/Summary.log

```
UefiSmbios -- PASS
ED2399AE-416E-46C6-ACA2-D9B1CF9975CD
/home/edhcha01/FWTS_NEW/arm-enterprise-acs/luv/build/tmp/work/aarch64-oe-linux/sbbr/v1.1+gitAUTOINC+b558bad254-r0/edk2/SctPkg/TestCase/UEFI/EFI/Generic/SbbrSmbios/BlackBoxTest/SbbrSmbiosBBTestFunction.c:199


Returned Status Code: Success

UefiSmbios: [PASSED]
  Passes........... 1
  Warnings......... 0
  Errors........... 0
```
